FluidNC/src/Motors/TMC2208Driver.cpp

```diff
@@ -12,19 +12,17 @@
 namespace MotorDrivers {
 
     void TMC2208Driver::init() {
-        if (!_uart_started) {
-            _uart->begin();
-            _uart->config_message("Trinamic", " Stepper ");
-            _uart_started = true;
+        TrinamicUartDriver::init();
+        if (!_uart) {
+            return;
         }
-
         if (_r_sense == 0) {
             _r_sense = TMC2208_RSENSE_DEFAULT;
         }
 
         tmc2208 = new TMC2209Stepper(_uart, _r_sense, _addr);
```

Comment:
Old Code but looks like a c&p error to me.
If I get it right it does not matter at all - but should be changed probably.